files/bin/deploy-iqe-cji.py

```diff
@@ -66,11 +66,24 @@ def build_number(self) -> str:
         try:
             build_number = check_run_id[:5]
         except TypeError:
-            display("There was a probelem with {check_run_id=}. Using default value of 1.")
+            display("There was a problem with {check_run_id=}. Using default value of 1.")
             build_number = "1"
 
         return build_number
 
+    @cached_property
+    def build_url(self) -> str:
+        """Create a build URL for the pipeline run"""
+
+        root_url = "https://console.redhat.com/application-pipeline/workspaces/cost-mgmt-dev/applications/"
+        params_url = "{component_name}/pipelineruns/{pipeline_run_name}"
+
+        if self.component_name and self.pipeline_run_name:
+            return f"{root_url}{params_url}".format(component_name=self.component_name, pipeline_run_name=self.pipeline_run_name)
+
+        print("Unable to create build URL. Missing component_name or pipeline_run_name. Returning root URL.")
+        return root_url
```

Comment:
This should return application name and pipeline run name from the labels. The problem with `component_name` is that is an arbitrary value that comes from the integration test. It could be incorrect.

We should get the value from `appstudio.openshift.io/application` and add that to the task environment and use it in the URL similar to [CHECK_RUN_ID](https://github.com/project-koku/koku-ci/blob/e39bad9af3ea44b861d91f4044b3ab515159e9e2/tasks/run-iqe-cji.yaml#L164-L172).

This should use `display` instead of `print`.

```suggestion
        application = os.environ.get("APPLICATION")
        return "https://console.redhat.com/application-pipeline/workspaces/cost-mgmt-dev/applications/{application}/{self.pipeline_run_name}"

```